linebot/api.py

```diff
@@ -124,6 +124,221 @@ def push_message(self, to, messages, timeout=None):
             '/v2/bot/message/push', data=json.dumps(data), timeout=timeout
         )
 
+    def get_rich_menu(self, rich_menu_id, timeout=None):
+        """Call get rich menu API.
+
+        https://developers.line.me/en/docs/messaging-api/reference/#get-rich-menu
+
+        Get rich menu object through a given rich_menu_id
+
+        :param str rich_menu_id: ID of the rich menu
+        :param timeout: (optional) How long to wait for the server
+            to send data before giving up, as a float,
+            or a (connect timeout, read timeout) float tuple.
+            Default is self.http_client.timeout
+        :type timeout: float | tuple(float, float)
+        :return: RichMenu instance
+        :type RichMenu: T <= :py:class:`linebot.models.reponse.RichMenu`
+        """
+        response = self._get(
+            '/v2/bot/richmenu/{rich_menu_id}'.format(rich_menu_id=rich_menu_id),
+            timeout=timeout
+        )
+
+        return RichMenu.new_from_json_dict(response.json)
+
+    def delete_rich_menu(self, rich_menu_id, timeout=None):
+        """Call delete rich menu API.
+
+        https://developers.line.me/en/docs/messaging-api/reference/#delete-rich-menu
+
+        Delete rich menu object through a given rich_menu_id
+
+        :param str rich_menu_id: ID of the rich menu
+        :param timeout: (optional) How long to wait for the server
+            to send data before giving up, as a float,
+            or a (connect timeout, read timeout) float tuple.
+            Default is self.http_client.timeout
+        :type timeout: float | tuple(float, float)
+        :return: An empty RichMenu instance
+        :type RichMenu: T <= :py:class:`linebot.models.reponse.RichMenu`
+        """
+        response = self._delete(
+            '/v2/bot/richmenu/{rich_menu_id}'.format(rich_menu_id=rich_menu_id),
+            timeout=timeout
+        )
+
+        return RichMenu.new_from_json_dict(response.json)
+
+    def create_rich_menu(self, data, timeout=None):
```

Comment:
Could you change to `rich_menu` instead of `data` ?
`data` is somewhat abstract.